modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Step.scala

```diff
@@ -23,31 +23,40 @@ final case class Step(
   config: StepConfig,
   resources: Set[Resource],
   breakpoint: Boolean,
-  skip: Boolean,
+  skipped: Boolean,
+  skipMark: Boolean,
   executions: List[List[Action]]
 )
 
 object Step {
 
   type Id = Int
 
+  def step(id: Int,
```

Comment:
This is a bit strange. Wouldn't it be better to call this `apply`? Or a more descriptive name

Reply:
The point is to create a `Step` with default values for the flags. Maybe I should call it `init`?